lib/flutter_map.dart

```diff
@@ -14,6 +14,7 @@ import 'package:latlong/latlong.dart';
 export 'src/plugins/plugin.dart';
 export 'src/layer/layer.dart';
 export 'src/layer/tile_layer.dart';
+export 'src/layer/mbitles/mbtiles_image_provider.dart';
```

Comment:
There's a typo here:  "mbitles" should be "mbtiles"

Reply:
Whoopsie, thanks for reviewing it :male_detective: